sbt/sbt-openapi-codegen/src/main/scala/OpenapiCodegenKeys.scala

```diff
@@ -0,0 +1,7 @@
+import sbt._
+
+trait OpenapiCodegenKeys {
+  lazy val swaggerFile = settingKey[File]("swagger file with the definitions")
```

Comment:
can't remember right now, but is there a convention for source generators and naming the setting keys? At first sight I would expect sth like `targetSwaggerFile`, but maybe that's not what happens in other projects